src/config.js

```diff
@@ -91,6 +116,8 @@ export function buildConfig (options: ConfigOptions = {}): Config {
   let defaultCacheDir = process.platform === 'darwin' ? path.join(config.home, 'Library', 'Caches') : null
   config.cacheDir = config.cacheDir || dir(config, 'cache', defaultCacheDir)
   config._version = '1'
+  userConfig = loadUserConfig(config.configDir)
+  config.skipAnalytics = skipAnalytics(userConfig)
```

Comment:
@KarateCowboy move the userConfig above the defaults and make a local variable and remove the assignment.  The reason is that if someone passes in `skipAnalytics` to `buildConfig` then this will overwrite what they pass rather than being the default